LiskHQ/lisk
# Merge development into feature branch - Closes #3084

### What was the problem?

development not merged into feature/improve_transactions_processing_efficiency

### How did I fix it?

By solving conflicts

### How to test it?

Review solved conflicts:

> CONFLICT (content): Merge conflict in framework/test/mocha/unit/modules/chain/submodules/rounds.js
CONFLICT (content): Merge conflict in framework/test/mocha/unit/modules/chain/submodules/blocks/verify.js
CONFLICT (content): Merge conflict in framework/test/mocha/unit/modules/chain/logic/transaction_pool.js
CONFLICT (content): Merge conflict in framework/test/mocha/integration/common.js
CONFLICT (content): Merge conflict in framework/test/mocha/integration/blocks/verify/verify.js
CONFLICT (content): Merge conflict in framework/test/mocha/common/application.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/submodules/rounds.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/submodules/loader.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/submodules/blocks/verify.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/submodules/blocks/process.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/submodules/blocks/chain.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/logic/transaction_pool.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/logic/transaction.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/init_steps/init_modules.js
CONFLICT (content): Merge conflict in framework/src/modules/chain/init_steps/init_logic_structs.js



### Review checklist

* The PR resolves #3084
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
